src/olympia/accounts/views.py

```diff
@@ -852,7 +852,11 @@ def get_jwt_payload(self, request):
         return authenticated_jwt
 
     def process_event(self, uid, event_key, event_data):
-        timestamp = event_data.get('changeTime') or datetime.now().timestamp()
+        timestamp = (
```

Comment:
Not new, but do we want to `int()` that to be consistent ? `time.time()` (and `datetime.now().timestamp()`) returns a float, not an integer.

Reply:
imo a float is better - we get millisecond precision when comparing the datetimes.  It's more consistent now we're doing `xxxx / 1000` as that'll be a float too.